document_page_approval_levels/views/document_page_view.xml

```diff
@@ -6,6 +6,38 @@
             <field name="model">document.page</field>
             <field name="inherit_id" ref="document_page.view_wiki_form"/>
             <field name="arch" type="xml">
+                <xpath expr="//field[@name='name']" position="after">
+                    <div class="oe_button_box oe_right">
+                        <field name="content_can_print" invisible="1"/>
+                        <button name="%(lodi_document.lodi_document_page_report)d"
```

Comment:
I call `lodi_document.lodi_document_page_report` because the pdf report is already declared there, so i didn't want to declare an extra report on this module. But i see now that the dependency is missing.
@dsabrinarg i can declare an extra report on this module to avoid modifying the `lodi_document` one, what do you think?